hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/infoschema/MappingsTable.java

```diff
@@ -51,13 +51,15 @@ public class MappingsTable extends InfoSchemaTable {
     private final String mappingsSchema;
     private final Collection<Mapping> mappings;
     private final Function<String, String> dataConnectionTypeResolver;
+    private final boolean securityEnabled;
```

Comment:
I was thinking of passing HazelcastInstance to the MappingsTable, but this way it is even simpler. We will probably refactor it anyway in 5.4